Meta/Lagom/Tools/CodeGenerators/LibTextCodec/GenerateEncodingIndexes.cpp

```diff
@@ -175,6 +176,7 @@ namespace TextCodec {
     TRY(file.write_until_depleted(generator.as_string_view().bytes()));
     return {};
 }
+} // end anonymous namespace
```

Comment:
It's not *terribly* prevalent, but it's mentioned (as `// namespace $thing`, not prescribed, though) in the coding sytle.

I think it's very helpful to avoid confusion when a namespace is open only in a part of the file. Time to make precedent :)